clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestCloneVeniceZKPaths.java

```diff
@@ -0,0 +1,174 @@
+package com.linkedin.venice;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class TestCloneVeniceZKPaths {
+  private final Set<String> CLUSTERS = new HashSet<>(Arrays.asList("cluster1", "cluster2"));
+  private final String BASE_PATH = "/venice-parent";
+
+  @Test
+  public void mainTest() {
+    testBuildRequiredPathsTree();
+    testGetVenicePaths();
+    testExtractVenicePaths();
+    // testCloneVenicePaths();
+  }
+
+  public void testCloneVenicePaths() {
+    // TODO write tests for cloneVenicePaths()
+  }
+
+  // tests AdminTool.extractVeniceZKPathsFromFile() as well as CloneVeniceZKPaths.extractVenicePaths()
+  public void testExtractVenicePaths() {
+    // change this variable to path of where Venice project is stored
+    String directory = "/Users/tonchen/Desktop/";
```

Comment:
You should put the `.txt` files in the `resources` directory (`clients/venice-admin-tool/src/test/resources`). You can create the `resources` directory if it doesn't exist

After that, we can use this to get the file path through:
```
URL resource = TestKafkaTopicDumper.class.getClassLoader().getResource(fileName);
File resourceFile = Paths.get(resource.toURI()).toFile();
String filePath = resourceFile.getAbsolutePath();
```